resources/styles/molecules/accordion/BpAccordion.vue

```diff
@@ -35,45 +41,44 @@
             >
                 <slot />
             </div>
-        </transition>
-    </div>
+        </Transition>
+    </button>
 </template>
-<script>
-export default {
-    props: {
+
+<script setup>
+    import { ref } from "@vue/reactivity";
```

Comment:
Why from '@vue/reactivity' instead of 'vue'? It's not wrong, but we should be consistent and always do one of the other through Boilerplate, and I'm not sure which is better.